rules/do-you-know-how-to-claim-expense-reimbursements-2/rule.md

```diff
@@ -50,63 +41,40 @@ At SSW we find the greatest overall value for expense reimbursements is to use t
  **Employees:** 
 
 
+1. Open the Xero Expense mobile app and submit a photo of the receipt for reimbursement or filing. For digitally emailed receipts, you can either take a photo of the email and upload it as per usual or you can upload the PDF version of it within the app.
 
+![**Figure: Xero Expense app submit a photo**](2020-11-25_15-44-09.png)
 
-  1. Using the Xero Expense mobile app, submit a photo of the receipt. If the receipt was sent via email, you can still submit a photo or screenshot of the email receipt
-
-
-
-
-
-
-
-
-![](2020-11-25_15-44-09.png)
-
-
- **Figure: Xero Expense app submit a photo** 
-
-
+**Figure: Xero Expense app submit a photo**
```

Comment:
Please remove this line as the square brackets will make it show up on the live site.

Reply:
Done